cogs/welcome/welcome.py

```diff
@@ -74,18 +75,19 @@ async def on_member_remove(self, leaveMember: discord.Member):
     async def on_channel_remove(self, removedChannel: discord.TextChannel):
         guild = removedChannel.guild
         # the channel to post welcome stuff in
-        welcomeIDSet = await self.config.guild(guild).KEY_WELCOME_CHANNEL_SET()
-        welcomeID = await self.config.guild(guild).KEY_WELCOME_CHANNEL()
+        welcomeIDSet = await self.config.guild(guild).welcomeChannelSet()
+        welcomeID = await self.config.guild(guild).welcomeChannel()
         # the channel msg should point to
-        welcomeLinkIDSet = await self.config.guild(guild).KEY_WELCOME_CHANNEL_LINK_SET()
-        welcomeLinkID = await self.config.guild(guild).KEY_WELCOME_CHANNEL_LINK()
+        welcomeLinkIDSet = await self.config.guild(guild).welcomeChannel_LINK_SET()
+        welcomeLinkID = await self.config.guild(guild).welcomeChannel_LINK()
```

Comment:
Missed this earlier, what are these two? I don't see it in Config.

Reply:
Forgot to get rid of this earlier